styles.css

```diff
@@ -0,0 +1,92 @@
+.toolbar-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 48px;
+  padding: 0 12px;
+}
+
+#logo {
+  width: 76px;
+  height: 20px;
+  margin-top: 10px;
+}
+
+#nav-logo {
+  width: 16px;
+  height: 20px;
+  margin-right: 20px;
+}
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+  height: 604px;
+  top: 92px;
+}
+
+#name {
+  height: auto;
+  width: 80%;
+  margin-left: 24px;
+  margin-top: 22px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
```

Comment:
- [ ] To align with [HTML and CSS best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)s, 
- [ ] It is recommended to use class names instead of IDs in your codebase. IDs should generally be reserved for unique elements, while class names offer more flexibility and reusability. 
- [ ] By utilizing class names, you can adhere to best practices and ensure a more organized and maintainable codebase.🤔🤔

Reply:
Thank you for your suggestion, i know the difference between class and id, we have different sections here and every section was unique so that's why i have used id instead of class but i will do the required changes.